memcache/memcache.go

```diff
@@ -297,6 +297,33 @@ func (c *Client) FlushAll() error {
 	return c.selector.Each(c.flushAllFromAddr)
 }
 
+// Version returns the memcached version from all servers.
+func (c *Client) Version() (map[net.Addr]string, error) {
```

Comment:
Do people want this signature? I wonder if there should be a plain string return value as well for the case where they all match?

Or maybe it should only return (version string, err error) and it returns an error if any don't match, explaining in the error which have which versions?